src/sensors/src/Accelerometer.cpp

```diff
@@ -139,6 +138,20 @@ SensorType Accelerometer::getSensorType() const
     return ACCELEROMETER;
 }
 
+bool Accelerometer::getLinkSensorTransform(Transform& link_H_sensor) const
+{
+    if( this->pimpl->parent_link_index < 0)
```

Comment:
Well the problem is without a setted 'legal' parent link index, how would one know which is the reference frame to which the link / sensor transform is being computed?